part2/phonebook/src/components/PersonForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const PersonForm = (
+    {
+        addContact,
+        newName,
+        newNumber,
+        handleNameChange,
+        handleNumberChange }) => (
+
+        <form onSubmit={addContact}>
+            <div>
+                name: <input
+                    value={newName}
+                    onChange={handleNameChange} />
+            </div>
+            <div>
+                number: <input
+                    value={newNumber}
+                    onChange={handleNumberChange} />
+            </div>
+            <div>
+                <button type="submit">add</button>
+            </div>
+        </form>
+    )
```

Comment:
Šitādu nonsensu pats Prettier uztaisīja, jeb šis fails ir paspējis tapt saglabāts pirms Prettier ienesi? 😁

Reply:
Tā man ir ar Prettier. Vai Tu domā tieši šo daļu?

```
... = (
    {
        addContact,
        newName,
        newNumber,
        handleNameChange,
        handleNumberChange }) => (
...
```

Reply:
Visu. Tur kopumā fancy, bet nu varbūt tāds ir prettier defaults (pirmo reizi gan redzu), mēs projektos nedaudz pielāgojam Prettier mūsu stilam. Whatev, ja saki, ka Prettier, tad no more point in discussing for now.